product-service/getProduct.js

```diff
@@ -0,0 +1,29 @@
+const { Client } = require('pg');
+
+const { successResponse, notFoundResponse, unexpectedErrorResponse, validationErrorResponse } = require('helper');
+const { dbOptions } = require('./db/config');
+
+module.exports = async ( event ) => {
+    const client = new Client(dbOptions);
+    try {
+        await client.connect();
+        console.log('getProduct handler input:', event);
+        const { pathParameters } = event;
+        const { id } = pathParameters;
+        const { rows: products } = await client.query(`select products.id, products.title, products.price, products.description, stocks.count from products left join stocks on products.id = stocks.product_id where products.id='${id}';`);
```

Comment:
you can make this query easier. For example: `select id, title, price, description, s.count from products p left join stocks s on p.id = s.product_id where p.id='${id}`

Reply:
always use parameterized queries as it prevents possible sql injections https://node-postgres.com/features/queries

Reply:
you are right. I changed code to use parameterized quiery